Automattic/jetpack
# Add My Jetpack to the Jetpack plugin navbar

Adds this button in the Jetpack plugin navbar that goes to My Jetpack. 

![image](https://user-images.githubusercontent.com/7129409/158382121-8fdb8515-ef98-4150-8b29-769ae7192817.png)

#### Changes proposed in this Pull Request:

Adds the button to the nav. 

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


Adds another nav button that will be tracked on click. 

#### Testing instructions:






- Navigate to /wp-admin/admin.php?page=jetpack#/dashboard
- Click the button
- Does it take you there?

Also test whether it respects the filter. 
- add `add_filter( 'jetpack_my_jetpack_should_initialize', '__return_false' );`
- The nav item should not be shown.
